front/migrations/20240110_kill_duplicate_chunks.ts

```diff
@@ -0,0 +1,83 @@
+import { QdrantClient } from "@qdrant/js-client-rest";
+
+const { QDRANT_API_KEY, QDRANT_URL } = process.env;
+
+const BUGGY_COLLECTION_NAMES: string[] = [];
+
+const client = new QdrantClient({
+  url: QDRANT_URL,
+  apiKey: QDRANT_API_KEY,
+});
+
+async function run() {
+  const result = await client.getCollections();
+  const collections = result.collections;
+
+  console.log(`Processing ${collections.length} collections.`);
+  let i = 0;
+  for (const c of collections) {
+    // console.log(c);
+    const chunkHashes: { [key: string]: string | number } = {};
+    const duplicates: (string | number)[] = [];
+
+    if (BUGGY_COLLECTION_NAMES.includes(c.name)) {
+      console.log(`SKIPPING ${c.name}`);
+      continue;
+    }
+    let offset:
+      | number
+      | undefined
+      | null
+      | string
+      | Record<string, unknown> = 0;
+    while (offset !== null && offset !== undefined) {
+      const res = await client.scroll(c.name, {
+        // with_payload: true,
```

Comment:
did you comment just for testing and will re-enable later? (IMHO param necessary)

Reply:
might also benefit from  `with_vector: false` (doc unclear on the default)

Reply:
We need neither of them :+1: